[2023-09-06T05:14:43.778+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: filtering_customer_consumption_backup.check_rows manual__2023-09-06T05:14:24.102041+00:00 [queued]>
[2023-09-06T05:14:43.806+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: filtering_customer_consumption_backup.check_rows manual__2023-09-06T05:14:24.102041+00:00 [queued]>
[2023-09-06T05:14:43.807+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-09-06T05:14:43.808+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-09-06T05:14:43.809+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-09-06T05:14:43.859+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): check_rows> on 2023-09-06 05:14:24.102041+00:00
[2023-09-06T05:14:43.871+0000] {standard_task_runner.py:55} INFO - Started process 504 to run task
[2023-09-06T05:14:43.879+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'filtering_customer_consumption_backup', 'check_rows', 'manual__2023-09-06T05:14:24.102041+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/filtering_customer_consumption_backup.py', '--cfg-path', '/tmp/tmpajk2vhk2']
[2023-09-06T05:14:43.883+0000] {standard_task_runner.py:83} INFO - Job 11: Subtask check_rows
[2023-09-06T05:14:44.058+0000] {task_command.py:376} INFO - Running <TaskInstance: filtering_customer_consumption_backup.check_rows manual__2023-09-06T05:14:24.102041+00:00 [running]> on host 60d3bee26398
[2023-09-06T05:14:44.305+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ductv50
AIRFLOW_CTX_DAG_ID=filtering_customer_consumption_backup
AIRFLOW_CTX_TASK_ID=check_rows
AIRFLOW_CTX_EXECUTION_DATE=2023-09-06T05:14:24.102041+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-09-06T05:14:24.102041+00:00
[2023-09-06T05:14:44.332+0000] {base.py:71} INFO - Using connection ID 'Postgress_1' for task execution.
[2023-09-06T05:14:44.353+0000] {base.py:71} INFO - Using connection ID 'Postgress_1' for task execution.
[2023-09-06T05:14:44.376+0000] {base.py:71} INFO - Using connection ID 'Postgress_1' for task execution.
[2023-09-06T05:14:44.408+0000] {logging_mixin.py:120} INFO - Key: consumption_alcoholic_20230906, Value: {'DataFrame Count': 2235, 'Postgres Count': 2235, 'Match': True}
[2023-09-06T05:14:44.409+0000] {logging_mixin.py:120} INFO - Key: consumption_cereals_bakery_20230906, Value: {'DataFrame Count': 1490, 'Postgres Count': 1490, 'Match': True}
[2023-09-06T05:14:44.416+0000] {logging_mixin.py:120} INFO - Key: consumption_meats_poultry_20230906, Value: {'DataFrame Count': 2980, 'Postgres Count': 2980, 'Match': True}
[2023-09-06T05:14:44.417+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-09-06T05:14:44.436+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=filtering_customer_consumption_backup, task_id=check_rows, execution_date=20230906T051424, start_date=20230906T051443, end_date=20230906T051444
[2023-09-06T05:14:44.539+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-09-06T05:14:44.672+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
